cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/relational/StringExpression.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
```

Comment:
@DJSagarAhire  Adding this class as mentioned in the design doc: https://docs.google.com/document/d/1hYnP_jAday-RKI0TPkJX_r2iJeroZXZAZB8XWx1nTDI/edit?resourcekey=0-lS0CoG1Ji02whjnCwRZJ_w#. @fernst and I synced, and it was decided that we need this class in cdap, to construct SQL expression in BigQuerySQLEngine

Reply:
Got it. So does this replace the Expression / ExpressionFactory classes that are being added in google-cloud?